src/langevin/rbpf_numba.py

```
@@ -0,0 +1,505 @@
+    mu_J:          float,
+    sigma_J:       float,
+    mu0:           np.ndarray,
+    C0:            np.ndarray,
+    dt:            float,
+    rng:           np.random.Generator,
+) -> tuple[np.ndarray, np.ndarray, np.ndarray, float, np.ndarray]:
+    """Numba-compiled Rao-Blackwellized Particle Filter (2012 Paper §III-B).
+
+    Drop-in replacement for src.langevin.rbpf.run_rbpf with an identical
+    call signature and return type.  Uses Numba @njit compilation for the
+    inner T × N loop, enabling 1-minute frequency runs (~400 k bars).
```

Ruff (0.15.9)

[warning] Docstring contains ambiguous `×` (MULTIPLICATION SIGN). Did you mean `x` (LATIN SMALL LETTER X)?

(RUF002)